spec/contacts_spec.rb

```diff
@@ -1,8 +1,43 @@
 require 'contacts'
 
 describe Contacts do
-  it 'can send first name to contacts database'do
+  let (:window) {ConMan.new}
+
+  before(:all) do
+    Qt::Application.new(ARGV)
+  end
+
+  def find_widget(name)
+    window.children.find { |child| child.object_name == name }
+  end
+
+  def create_contact(first_name, last_name, dob, email, tel, address)
+    find_widget("first_name").setText("Priya")
+    find_widget("last_name").setText("Patil")
+    find_widget("dob").setText("04.10.1987")
+    find_widget("telephone").setText("123")
+    find_widget("email").setText("abc@me.com")
+    find_widget("Add").click
+  end
+
+  it 'can send name field to contacts database'do
+    create_contact("Priya", "Patil", "04.10.1987", "123", "abc@me.com", "2 Cedar Way")
     name = Contacts.find_by(first_name: 'Priya')
     expect(name.first_name).to eq 'Priya' 
   end
+
+  it 'can send last name field to contacts database' do
+    name = Contacts.find_by(last_name: 'Patil')
```

Comment:
are these not dependent on the test above to run? what if i were to randomize the order of test?
you need to make sure you call create_contact within each test. probably the best way to do that is with a context/before(:each) block.